### PR TITLE
style(list): fix no more items styling

### LINKED ISSUE #508: [Enhancement] For load on demand list the spinner and no-more-items text should be under the list container
**Is your enhancement request related to a problem? Please describe.**
For load on demand list, the spinner and no-more-items text should be under the list container

**Describe the solution you'd like**
For load on-demand list the spinner and no-more-items text should be under the list container

### PR DESCRIPTION
**Pull request by lydemann**
Closes #508 
Ensure loading is shown outside of list container. Also adjusted sizes and font for show more to make it look consistent between web/native.

Web:
image

Native:
image

### CODE REVIEW

File: src/kirby/components/list/list.component.html [hunk 1]
```diff
@@ -1,8 +1,5 @@
-<ion-list kirbyInfiniteScroll
-          (scrollEnd)="onLoadOnDemand()"
-          [disabled]="!isLoadOnDemandEnabled"
-          [class.has-header]="listHeaderTemplate"
-          [class.has-footer]="listFooterTemplate">
+<ion-list kirbyInfiniteScroll (scrollEnd)="onLoadOnDemand()" [disabled]="!isLoadOnDemandEnabled"
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.html]: Do we have a defined style rule, on how we position attributes? Inline or on newline. @jakobe or @hafstad any points on this?

Comment (jakobe) [reply src/kirby/components/list/list.component.html]: Don't think so, but in general I think we should keep the formatting and only change it if we decide to...

File: src/kirby/components/list/list.component.tns.html [hunk 2]
```diff
@@ -45,7 +45,7 @@
 
     <!-- FOOTER -->
     <ng-template tkListViewFooter *ngIf="!isLoadOnDemandEnabled && noMoreItemsText && !listFooterTemplate">
-      <Label class="footer" text="{{noMoreItemsText}}"></Label>
+        <Label class="show-more-items" text="{{noMoreItemsText}}"></Label>
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.tns.html]: A bit nitpicking here 😄 

But should the class not called something `no-more-items`  like it is on web?

**Comment by lydemann** [comment]
UI has been reviewed now by UX and it passed.

**Pull request merged**